signup.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="https://ragaui.netlify.app/Components.css" />
+    <link rel="stylesheet" href="signup.css" />
+    <title>Surplus</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar ">
+            <div class="navbar-logo">
+                <i class="fas fa-meteor nav-logo"></i>
+                <h2>Surplus</h2>
+            </div>
+            <div class="navbar-search">
+                <i class="fas fa-search search-icon"></i>
+                <input type="text" class="i-text" placeholder="Search for products, brands and more"/>
+            </div>
+            <div class="navbar-cart">
+                <button class="btn primary">Login</button>
+                    <a href="#"><i class="fas fa-heart nav-icon nav-icon-wishlist"></i> Wishlist</a>
+                    <a href="#"><i class="fas fa-cart-plus nav-icon"></i> Cart</a>
+                
+            </div>
+        </nav>
+    </header>
+    <main class="login-page-main">
+
+        <section class="login-card-container">
```

Comment:
Use a form element instead of the section element to create your signup or signing forms. This is semantically right .

Reply:
Yeah, made changes